superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -235,20 +236,21 @@ export default class ResultSet extends React.PureComponent {
           <Alert bsStyle="info">
             {t(object)} [
             <strong>
-              {tmpSchema}.{tmpTable}
+              {tempSchema ? `${tempSchema}.` : ''}
```

Comment:
When testing on sqllite, there wasn't always a schema, so i cleaned up the cases where schema can be null